ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -859,6 +859,209 @@ CMitsubaLoader::SContext::tex_ass_type CMitsubaLoader::getTexture(SContext& ctx,
 	return std::move(layer);
 }
 
+static bsdf::E_OPCODE BSDFtype2opcode(const CElementBSDF* bsdf)
```

Comment:
remind me again, why no different opcode for rough vs. non rough?

Reply:
hm, yea, you reminded me that i need one more bit for specifying smooth/lambertian. So it could be another bsdf type just as well.. i'll make separate opcodes for rough variants

Reply:
smooth vs. rough... lambertian is a whole different thing